pkg/reconcile/pipeline/handler/project/impl.go

```diff
@@ -154,7 +155,8 @@ func BindingsAsFiles(ctx pipeline.Context) {
 			Name: bindingName,
 			VolumeSource: corev1.VolumeSource{
 				Secret: &corev1.SecretVolumeSource{
-					SecretName: secretName,
+					SecretName:  secretName,
+					DefaultMode: &defaultModeValue,
```

Comment:
I'm concerned that this could brake some usecases where the user running the pod is not the user the owns the mount point. Is this a possible scenario today?